guide/naming.md

```diff
@@ -0,0 +1,58 @@
+# VDD Naming Convetions / Semantic Versioning
+
+In order to facilitate tracability over several iterations of English
+and TLA+ specifications, and to make clear to external readers what is
+the state of a given document, we use the following convention:
+
+## English Spec
+
+As part of the version, each English specification should have one
+of the following stages:
+
+```
+Draft --> Proposal --> Reviewed
+             |
+             |--> Published
+```
+
+
+- There should be an additional stage `Withdrawn`, for the uncommon
+  outcome that at some point we find the specification useless
+- `Draft` should live in auxiliary branches
+- `Proposal` should live in a PR
+- `Reviewed`, `Published`, `Withdrawn` should live in the main
+  (f.k.a. master) branch
+- From any stage, if updates need to be made, we first transition back
+  to Draft.
+  
+  
+Whenever the stage changes for a English specification, the version
+number should increase. Here an example trace
+
+Draft.1 -> Proposal.2 -> Reviewed.3 -> Draft.4 -> Proposal.5 ->
+Published.6
+
```

Comment:
An example filename and document title would help make this even clearer. 

For the title, would it look something like

```
Draft.1: Title of the Spec
```

or have a different format?

For the file name, would this be part of the extension of the file

```
title-of-the-spec.Draft.1.md
```

or a prefix to the file name?